src/commands/sbom/renderer.ts

```diff
@@ -21,8 +21,11 @@ export const renderInvalidFile = (sbomReport: string) => {
 
 export const renderInvalidPayload = (sbomReport: string) => {
   const reportPath = `[${chalk.bold.dim(sbomReport)}]`
+  let fullStr = ''
+  fullStr += chalk.red(`Cannot generate payload for file ${reportPath}.\n`)
+  fullStr += chalk.red(`Make sure you run the command inside a git repository and the SBOM file is valid\n`)
 
-  return chalk.red(`Cannot generate payload for file ${reportPath}.\n`)
+  return fullStr
```

Comment:
I think you can just do something like 
```js
  return chalk.red(
    `Cannot generate payload for file ${reportPath}.\n` +
    `Ensure that you run the command inside a git repository, and that the SBOM file is valid.\n`
  )
```

Unfortunately GH wont let me make suggestions when there's a deletion 😑